docs/modules/globalfee.md

```diff
@@ -111,63 +115,57 @@ Even though each node can set its own `min_gas_prices` and `bypass-min-fee-msg-t
 
+- **Case 1**: globalfee=[], min_gas_prices=0.0001uatom, gas=2000000
   This is the same case as globalfee=0uatom, min_gas_prices=0.0001uatom, gas=2000000.
   - paidfee = "2000000 * 0.0001uatom", pass
-  - paidfee = "2000000 * 0.0001uatom, 0stake", pass
   - paidfee = "2000000 * 0.0001uatom, 1stake", fail
   - paidfee = "2000000 * 0.0001/2uatom", fail
-  - paidfee = "", pass
+  - paidfee = "", fail
 
-- **Case 2**: globalfee=[], min_gas_prices="", gas=2000000
+- **Case 2**: globalfee=[], min_gas_prices="", gas=2000000 (When globalfee empty, the [default globalfee of 0uatom](https://github.com/cosmos/gaia/blob/d6d2933ede1aa1a13040f5aee2f0f7b795c168d0/x/globalfee/ante/fee.go#L135) will be used.)
   - paidfee = "", pass
-  - paidfee = "0uatom", pass
-  - paidfee = "1uatom", pass
-  - paidfee = "0uatom, 0stake", pass
-  - paidfee = "0uatom, 1stake", fail
+  - paidfee = "2000000 * 0.0001uatom", pass
+  - paidfee = "2000000 * 0.0001stake", fail
   
 - **Case 3**: globalfee=0.0002uatom, min_gas_prices=0.0001uatom, gas=2000000 (global fee is lower than min_as_price)
   - paidfee = "2000000 * 0.0002uatom", pass
   - paidfee = "2000000 * 0.0001uatom", fail
   - paidfee = "2000000 * 0.0002uatom, 1stake", fail
-  - paidfee = "2000000 * 0.0002uatom, 0stake", pass
   - paidfee = "2000000 * 0.0002stake", fail
   - paidfee = "", fail
-  - paidfee = 0uatom, fail
   
 - **Case 4**:  globalfee=0.0001uatom, min_gas_prices=0.0002uatom, gas=2000000 (global fee is higher than min_as_price)
   - paidfee = "2000000 * 0.0002uatom", pass
   - paidfee = "2000000 * 0.0001uatom", fail
   - paidfee = "2000000 * 0.0002uatom, 1stake", fail
-  - paidfee = "2000000 * 0.0002uatom, 0stake", pass
+  - paidfee = "2000000 * 0.0002uatom", pass
   - paidfee = "2000000 * 0.0002stake", fail
   - paidfee = "", fail
   - paidfee = 0uatom, fail
   
 - **Case 5**: globalfee=[0uatom, 1stake], min_gas_prices="", gas=200000.
-- fees="2000000 *0uatom, 2000000* 0.5stake", fail, (this is due to [fee parsing, zero coins are removed](https://github.com/cosmos/cosmos-sdk/blob/e716e4103e934344aa7be6dc9b5c453bdec5f225/client/tx/factory.go#L144), it equals to paidfees = 0.5stake in this case)
-- paidfees="", pass
-- paidfees="2000000 * 1uatom, 0.5stake", pass
-- paidfees="0uatom, 0stake" pass, (due to the parsing of paidfees, it makes paidfees="")
-- paidfees="2000000 * 1stake", pass
+  - paidfees="2000000 * 0.5stake", fail
+  - paidfees="", pass
```

Comment:
The global fees acts as a base upon which (local) min fees act as further tightening on that base. Given that the base (global fees) stipulates any atom and >= 1stake is required, `""` should FAIL.